Emission.podspec

```diff
@@ -16,9 +17,7 @@ podspec = Pod::Spec.new do |s|
   s.summary        = 'React Native Components used by Eigen.'
   s.homepage       = 'https://github.com/artsy/emission'
   s.license        = 'MIT'
-  s.author         = { 'Eloy Durán' => 'eloy@artsy.net',
-                       'Maxim Cramer' => 'maxim@artsy.net',
-                       'Sarah Scott' => 'sarah.scott@artsy.net' }
+  s.author         = { 'Artsy Mobile' => 'mobile@artsy.net' }
```

Comment:
`/shrug` I was already making changes to the Podspec, so maybe this makes sense?